scripts/provisioning/roles/motr-build/vars/RedHat.yml

```diff
@@ -50,7 +50,7 @@ motr_build_deps_pkgs:
   - rpm-build
   - systemd-devel
   - libedit-devel
-  - 'https://www.nasm.us/pub/nasm/releasebuilds/2.14/linux/nasm-2.14-0.fc27.x86_64.rpm'
+  - isa-l
```

Comment:
Do we have to add the seagate website ( http://cortx-storage.colo.seagate.com/releases/cortx/github/main/centos-7.8.2003/754/prod/3rd_party/motr_uploads/) as a repo?